include/hpp/fcl/narrowphase/gjk.h

```diff
@@ -150,13 +150,25 @@ struct HPP_FCL_DLLAPI GJK {
 
   typedef unsigned char vertex_id_t;
 
+  /// @brief A simplex is a set of up to 4 vertices.
+  /// Its rank is the number of vertices it contains.
+  /// @note This data structure does **not** own the vertices it refers to.
+  /// To be efficient, the constructor of `GJK` creates storage for 4 vertices.
+  /// Since GJK does not need any more storage, it reuses these vertices
+  /// throughout the algorithm by using multiple instance of this `Simplex`
+  /// class.
   struct HPP_FCL_DLLAPI Simplex {
     /// @brief simplex vertex
     SimplexV* vertex[4];
     /// @brief size of simplex (number of vertices)
     vertex_id_t rank;
 
     Simplex() {}
+
+    inline void reset() {
```

Comment:
```suggestion
    void reset() {
```

Methods defined in a class/struct are inline by default.